vra/resource_network_ip_range.go

```diff
@@ -0,0 +1,196 @@
+package vra
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/vmware/vra-sdk-go/pkg/client/network_ip_range"
+	"github.com/vmware/vra-sdk-go/pkg/models"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceNetworkIPRange() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNetworkIPRangeCreate,
+		Read:   resourceNetworkIPRangeRead,
+		Update: resourceNetworkIPRangeUpdate,
+		Delete: resourceNetworkIPRangeDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"fabric_network_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"start_ip_address": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				// Do we need to validate?
+			},
+			"end_ip_address": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				// Do we need to validate?
+			},
+			"ip_version": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"IPv4", "IPv6"}, true),
+			},
+			"progress": &schema.Schema{
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
```

Comment:
`progress` is an attribute from RequestTracker and is not really part of the networkIPRange model. You may omit this.